x/ibc/testing/coordinator.go

```diff
@@ -144,70 +139,178 @@ func (coord *Coordinator) UpdateClient(
 }
 
 // CreateConnection constructs and executes connection handshake messages in order to create
-// OPEN channels on source and counterparty chains. The connection information of the source
-// and counterparty's are returned within a TestConnection struct. If there is a fault in
-// the connection handshake then an error is returned.
-//
-// NOTE: The counterparty testing connection will be created even if it is not created in the
-// application state.
+// OPEN channels on chainA and chainB. The connection information of for chainA and chainB
+// are returned within a TestConnection struct. The function expects the connections to be
+// successfully opened otherwise testing will fail.
 func (coord *Coordinator) CreateConnection(
-	sourceID, counterpartyID,
-	clientID, counterpartyClientID string,
-	state connectiontypes.State,
-) (TestConnection, TestConnection, error) {
-	source := coord.GetChain(sourceID)
-	counterparty := coord.GetChain(counterpartyID)
+	chainA, chainB *TestChain,
```

Comment:
I originally had it that way, but my reasoning is any function which does the same functions on the first chain as on the second chain (creating open channels, connections etc) would use `chainA, chainB` and any function that is done from the perspective of one chain uses `source` `counterparty` (chanOpenInit, createclient, etc)

Reply:
Let me know if you'd still like me to update it this. 

To put things more concisely, using source and counterparty in these funcs gets confusing because then for example upon calling `ChanOpenTry`, source becomes counterparty and counterparty becomes source (ie their naming swaps).

Reply:
it's ok, let's leave it as is and add a note